src/server/index.js

```diff
@@ -76,7 +77,22 @@ const configureRoutes = (app) => {
     require('./routes/changelog.js')(app);
 };
 
+const trackUrlTelemetry = (req, res, next) => {
+    var props = {
+        url: req.url
+    };
+
+    if(req.query['source'] !== undefined) {
+        props.source = req.query['source'];
+    }
+
+    telemetry.trackEvent('online-activity-url', props);
+    next();
+}
+
 const configureFallbacks = (app) => {
+    app.use(trackUrlTelemetry);
```

Comment:
IIRC, this will only send telemetry for static assets. Is this what you want?

Reply:
@molant Yes, that should be okay. We are mainly interested in tracking the documentation urls.

Reply:
We already have google analytics and can get this information (and more) from there. Is the plan to remove GA?

Reply:
@molant Not sure about GA, we can remove if it is redundant. This is to mainly to track referrals (like when "learn more" is clicked from DevTools panel). Having it in AI lets us pull it for internal reporting as well.